avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/PartitionIdGeneration.scala

```diff
@@ -0,0 +1,45 @@
+package com.linkedin.avro2tf.jobs
+
+import com.linkedin.avro2tf.parsers.TensorizeInParams
+import com.linkedin.avro2tf.utils.{Constants, HashingUtils}
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.functions.{expr, udf}
+
+/**
+ * Append a partition id according to the specified partition field and num of partitions
+ */
+object PartitionIdGeneration {
+
+  val globalSalt: String = new scala.util.Random(888777666).alphanumeric.take(16).mkString
```

Comment:
Do we want to make `888777666` and `15` constants and/or explain them in comments for clarification?

Reply:
sure, i will move them to constants